seed/data_importer/tests/integration/test_data_import.py

```diff
@@ -154,8 +154,8 @@ def setUp(self):
         self.maxDiff = None
 
         # setup the ImportFile for using the example zip file
-        filename = 'example-bsync-single.zip'
-        filepath = osp.join(osp.dirname(__file__), '..', 'data', filename)
+        filename = 'ex_1_and_buildingsync_ex01_measures.zip'
+        filepath = osp.join(BASE_DIR, 'seed', 'building_sync', 'tests', 'data', filename)
```

Comment:
If `BASE_DIR` is the best approach to manage dirs, then we should update this everywhere. But not now.